rules.adoc

```diff
@@ -767,21 +767,23 @@ robot’s dimensions must not exceed the following limits:
 
 |===
 |sub-league | *Soccer* *Open* | *Soccer Lightweight*
-|size / (robot must fit {++smoothly++} into a cylinder of this diameter) | 18.0 cm | 22.0 cm +
+|size ^[0]^  | 18.0 cm | 22.0 cm +
```

Comment:
@dbscoach I moved this out because it didn't look good -- I hope you understand :slightly_smiling_face:

Reply:
Makes a lot of sense, was way fewer words when it went in.